Development/nmos/test/configuration_utils_test.cpp

```diff
@@ -1680,3 +1681,80 @@ BST_TEST_CASE(testModifyRebuildableBlock)
         }
     }
 }
+
```

Comment:
Should this test not live in `control_protocol_utils_test.cpp`?

Reply:
Very good point